scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -379,6 +379,37 @@ export function SubscribersDirective(
                     ));
                 }
             };
+
+            /**
+             * Function to generate and set appropriate schedule/label messages
+            */
+            $scope.getScheduleOrStatusLabel = function(subscriber) {
+                if (!subscriber) return '';
+
+                const hasSchedule = subscriber.schedule?.start_date && subscriber.schedule?.end_date;
```

Comment:
Do not implicitly cast to boolean. Use:

`const hasSchedule = subscriber.schedule?.start_date != null && subscriber.schedule?.end_date != null;`